elastic/kibana
# Speed up enqueueing of rule actions

In this PR, I'm making the bulk enqueueing of rule actions faster by avoiding to wait for an index refresh when creating the `action_task_params` saved objects. The removal of waiting for a refresh speeds up the rule executions by ~500ms when there are actions to enqueue for alerts.

## To verify
1. Create an alerting rule that fires actions
2. Notice the actions are running successfully

You can also compare run times with main when doing your verification steps. A simple index threshold firing one action would go from 100ms-1000ms to a more consistent 100ms runtime..

## Note to reviewer(s)

It's easier to review the test changes if you append `?w=1` to the URL when looking at the file diff. It will remove whitespace differences and show fewer lines changed.
